### ISSUE TITLE
Interpreter issues in Linux

### ISSUE THREAD

**Issue by LyesSaadi** [azizamari/Ncodi#54]
Hello,

When trying this on Linux, there are some issues with the interpreter not reprinting lines in the same place if it makes sense. It also retains "returns" until the end of the program. Let me illustrate this:

![image](https://user-images.githubusercontent.com/33597258/126405134-05547294-5d5e-4386-a9aa-cd619633870f.png)

Except for that, it compiles wonderfully.

I have no knowledge in C#, so I cannot help unfortunately...

**Comment by azizamari** [comment]
That's true I experienced this before when testing but forgot to fix it. Btw you can run the Ncodi.Cli project then you can run code from files with .ncodi extension using this command (./Ncodi.Cli run filename.ncodi) in the folder where it got built I am not sure of the path but it somewhere in src/Ncodi.Cli/bin
